packages/react-native/src/primitives/TextField/__tests__/TextField.spec.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { fireEvent, render } from '@testing-library/react-native';
+import TextField from '../TextField';
+
+const placeHolderText = 'Placeholder';
+const labelText = 'Label';
+const testID = 'textInput';
+const defaultProps = {
+  testID: testID,
+  label: labelText,
+  placeholder: placeHolderText,
+};
+
+const onChangeText = jest.fn();
+
+describe('TextField', () => {
+  it('renders as expected', async () => {
+    const { toJSON, findAllByRole, getByTestId, getByText } = render(
+      <TextField {...defaultProps} />
+    );
+    expect(toJSON()).toMatchSnapshot();
+
+    expect(await findAllByRole('text')).toHaveLength(1);
+    const textInput = getByTestId(testID);
+    expect(textInput.props.editable).toBe(true);
+    expect(textInput.props.secureTextEntry).toBeFalsy();
```

Comment:
```suggestion
    expect(textInput.props.secureTextEntry).toBe(false);
```

Reply:
This is undefined when not passed in, not false.